hazelcast/src/main/java/com/hazelcast/map/impl/record/RecordStatistics.java

```diff
@@ -16,78 +16,34 @@
 
 package com.hazelcast.map.impl.record;
 
-import com.hazelcast.nio.ObjectDataInput;
-import com.hazelcast.nio.ObjectDataOutput;
 import com.hazelcast.nio.serialization.DataSerializable;
-import com.hazelcast.util.Clock;
-import java.io.IOException;
 
 /**
- * TODO empty statistics.
- * Some statistics of a {@link com.hazelcast.map.impl.record.Record}
+ * Represents statistics of a {@link Record}.
  */
-public class RecordStatistics implements DataSerializable {
+public interface RecordStatistics extends DataSerializable {
 
-    // TODO is volatile needed? if yes then hits should be atomicnumber
-    protected int hits;
-    protected long lastStoredTime;
-    protected long expirationTime;
+    /**
+     * Used instead of null.
+     */
+    RecordStatistics EMPTY_STATS = new EmptyRecordStatistics();
```

Comment:
I would not define this singleton here, I would define it on the EmptyRecordStatistics. And call it INSTANCE.